packages/installWatson.sh

```diff
@@ -11,14 +11,22 @@ source "$PACKAGE_HOME/util.sh"
 
 echo Installing Watson package.
 
-createPackage watson \
-    -a description "Actions for the Watson analytics APIs" \
-    -a parameters '[ {"name":"username", "required":false}, {"name":"password", "required":false, "type":"password"} ]'
+createPackage watson-translate \
```

Comment:
name this one `watson-translator` to match the new service name.
The `translate` service is deprecated